simulation_parameters/Constants.cs

```diff
@@ -183,25 +183,25 @@ public static class Constants
     /// </remarks>
     public const float BASE_MOVEMENT_ATP_COST = 1.0f;
 
-    public const float FLAGELLA_ENERGY_COST = 4.0f;
+    public const float FLAGELLA_ENERGY_COST = 8.0f;
 
-    public const float FLAGELLA_BASE_FORCE = 20.0f;
+    public const float FLAGELLA_BASE_FORCE = 50.0f;
 
-    public const float BASE_MOVEMENT_FORCE = 1000.0f;
+    public const float BASE_MOVEMENT_FORCE = 650.0f;
```

Comment:
I'm not exactly sure I like how sluggish the single cytoplasm gets after this...